packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -271,7 +271,7 @@ function Tab({
           className={cx(
             tabStyles,
             themeClass,
-            isNonExistent && nonExistentStyles,
+            isGhostNamespace && nonExistentStyles,
```

Comment:
While you are at it, `nonExistentStyles` -> `ghostNamespaceStyles` or something?